test/src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -501,7 +506,19 @@ protected Hudson newHudson() throws Exception {
         File home = homeLoader.allocate();
         for (Runner r : recipes)
             r.decorateHome(this,home);
-        return new Hudson(home, createWebServer(), useLocalPluginManager ? null : pluginManager);
+        if (useLocalPluginManager) {
+            pluginManager = new LocalPluginManager(home) {
+                @Override
+                protected Collection<String> loadBundledPlugins() {
+                    // Overriding so we can force loading of the detached plugins for testing
```

Comment:
I guess I should ask—is this something you are doing in order that a newly added test can verify detached plugin loading; or something you are doing because otherwise existing tests (esp. those with `@LocalData`) will start breaking?